base_kanban_stage/tests/test_base_kanban_abstract.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models
+from odoo.tests.common import SavepointCase
+
+
+class BaseKanbanAbstractTester(models.TransientModel):
+    _name = 'base.kanban.abstract.tester'
+    _inherit = 'base.kanban.abstract'
+
+
+class TestBaseKanbanAbstract(SavepointCase):
+
+    @classmethod
+    def _init_test_model(cls, model_cls):
+        """ It builds a model from model_cls in order to test abstract models.
+
+        Note that this does not actually create a table in the database, so
+        there may be some unidentified edge cases.
+
+        Args:
+            model_cls (openerp.models.BaseModel): Class of model to initialize
+        Returns:
+            model_cls: Instance
+        """
```

Comment:
This came from template. Added note to PR https://github.com/OCA/maintainer-tools/pull/242/files#r98305473